src/bakery_manager_mobile/lib/assets/constants.dart

```diff
@@ -3,9 +3,24 @@ const String recipePageRoute = '/recipes';
 const String ingredientPageRoute = '/ingredients';
 const String productPageRoute = '/products';
 const String taskPageRoute = '/tasks';
-const String salesPageRoute = '/sales';
+const String taskDetailsPageRoute = '/tasks/details';
 const String loginPageRoute = '/login';
 const String registrationPageRoute = '/login/registration';
 const String addRecipePageRoute = '/recipes/addRecipe';
+const String editRecipePageRoute = '/recipes/editRecipe';
 const String recipeDetailsPageRoute = '/recipes/details';
 const String myAccountRoute = '/myaccount';
+const String ingredientDetailsPageRoute = "/ingredients/details";
+const String editAccountPageRoute = "/myaccount/edit";
+final List<String> recipeCatagories = [
```

Comment:
Are categories finalized?

Reply:
No though these can be changed easily if needed.